runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/BatchModeExecutionContext.java

```diff
@@ -509,7 +508,7 @@ public Long extractThrottleTime() {
       CounterCell throttlingMsecs =
           container.tryGetCounter(DataflowSystemMetrics.THROTTLING_MSECS_METRIC_NAME);
       if (throttlingMsecs != null) {
-        totalThrottleTime += TimeUnit.MICROSECONDS.toSeconds(throttlingMsecs.getCumulative());
```

Comment:
Can we stick with the TimeUnit class and use TimeUnit.MICROSECONDS.toMillis(throttlingMsecs.getCumulative()) since it helps readability.

Reply:
`toMilllis()` does not round it. The thought was it might be bit more accurate when minimal throttled times are in milliseconds rather than seconds. I will change it.

Reply:
Done.